### ISSUE TITLE
[BUG] symbol ENCRYPTED_PGP is not applied to some encrypted PGP mails

### ISSUE THREAD

**Issue by heraklit256** [rspamd/rspamd#3349]
### Prerequisites

* [X] Put an X between the brackets on this line if you have done all of the following:
    * Read about bug reporting in general: https://rspamd.com/doc/faq.html#how-to-report-bugs-found-in-rspamd
    * Enabled relevant debugging logs: https://rspamd.com/doc/faq.html#how-to-debug-some-module-in-rspamd 
    * Checked the FAQs about Core files in case of fatal crash: https://rspamd.com/doc/faq.html#how-to-figure-out-why-rspamd-process-crashed
    * Tried ASAN package and obtained the ASAN report (if possible): https://rspamd.com/doc/faq.html#asan-builds 
    * Checked that your issue isn't already filed: https://github.com/issues?utf8=%E2%9C%93&q=is%3Aissue+user%3Arspamd
    * Checked that there is not already an experimental package or master branch

**Describe the bug**
PGP encrypted mails as emitted by Thunderbird using [Enigmail](https://enigmail.net/index.php/en/) are not detected as being `ENCRYPTED_PGP` by rspamd. However, PGP encrypted mails as emitted by Mutt are, while they seem to have very little difference.

Sample of a PGP encrypted mail as emitted by Thunderbird using Enigmail:
```
Content-Type: multipart/encrypted;
 protocol="application/pgp-encrypted";
 boundary="ZLWRaggnXh0KVFSwp017TD2fbHmQ9jtyd"

This is an OpenPGP/MIME encrypted message (RFC 4880 and 3156)
--ZLWRaggnXh0KVFSwp017TD2fbHmQ9jtyd
Content-Type: application/pgp-encrypted
Content-Description: PGP/MIME version identification

Version: 1

--ZLWRaggnXh0KVFSwp017TD2fbHmQ9jtyd
Content-Type: application/octet-stream; name="encrypted.asc"
Content-Description: OpenPGP encrypted message
Content-Disposition: inline; filename="encrypted.asc"

-----BEGIN PGP MESSAGE-----

hF4DLLcf6EDug/wSAQdA5dOaysfrEjHhzArAWLBaLX5Oi5IfYZN4rXD9XQ9UlVQw
[etc., etc., etc.]
-----END PGP MESSAGE-----

--ZLWRaggnXh0KVFSwp017TD2fbHmQ9jtyd--
```

Sample of a PGP encrypted mail as emitted by Mutt:
```
Content-Type: multipart/encrypted;
 protocol="application/pgp-encrypted";
 boundary="BCj5OkZPIU+vLR1l"
Content-Disposition: inline

--BCj5OkZPIU+vLR1l
Content-Type: application/pgp-encrypted
Content-Disposition: attachment

Version: 1

--BCj5OkZPIU+vLR1l
Content-Type: application/octet-stream
Content-Disposition: attachment; filename="msg.asc"

-----BEGIN PGP MESSAGE-----

hQIMA3bM6VmvUYPDAQ/7BGm/+ukg+PJ2PY67jiuTeo7Qt05fUCgdA5UI+QgSrlDb
[etc., etc., etc.]
-----END PGP MESSAGE-----

--BCj5OkZPIU+vLR1l--
```

### Steps to Reproduce

1. Compose a message in Thunderbird.
2. Encrypt it using the Enigmail addon and send it.
3. Once rspamd has processed it, it does not get the `ENCRYPTED_PGP` symbol.

**Expected behavior**
Encrypted PGP messages are always tagged with `ENCRYPTED_PGP`.

### Versions
```
user@machine:~> rspamd --version
Rspamd daemon version 2.5
```
### Additional Information

Thank you for having a look at this!

**Comment by heraklit256** [comment]
@vstakhov: Thank you! :-)

**Comment by heraklit256** [comment]
Although this bug is still valid, I expect it to be fixed in the next release. Until that, keeping this opened makes more sense to me.

**Comment by heraklit256** [comment]
- ping -

**Comment by heraklit256** [comment]
This issue has been fixed in rspamd 2.6.
